src/themes/Mikings/SoftClose/SoftClose.scss

```diff
@@ -46,7 +46,6 @@
 			background-color: colors.$secondary;
 			border-radius: 18px;
 			font-size: 14px;
-			line-height: 40px;
```

Comment:
Isn't removing these line-heights a BC-breaking change for the archived banners?
I'm not sure what we should do in cases like these.

Reply:
We had issues that the line height was breaking the text alignment on the buttons, this fixes it. So it'll now also be fixed in the archived banners, which I think is fine. We always have the tags in the repo if we need an exact global state.